metadata-ingestion/examples/library/lineage_emitter_view_job_mcp.py

```diff
@@ -0,0 +1,70 @@
+import datahub.emitter.mce_builder as builder
+import json
+
+from datahub.emitter.mcp import MetadataChangeProposalWrapper
+from datahub.emitter.rest_emitter import DatahubRestEmitter
+from datahub.metadata.com.linkedin.pegasus2avro.dataset import (
+    FieldLineageDetailsClass,
+    DatasetFieldsLineages
+)
+from datahub.metadata.schema_classes import ChangeTypeClass, DataJobInputOutputClass
+
+def datasetUrn(tbl):
+    return builder.make_dataset_urn("postgres", tbl)
+
+def fldUrn(tbl, fld):
+    return f"urn:li:schemaField:({datasetUrn(tbl)}, {fld})"
```

Comment:
Can we add a `make_dataset_field_urn` utility function to the `datahub.emitter.mce_builder` instead of hard-coding the urn strings here?